src/main/kotlin/app/revanced/patches/all/screenshot/removerestriction/patch/RemoveScreenshotRestrictionPatch.kt

```diff
@@ -17,7 +17,7 @@ import org.jf.dexlib2.iface.reference.MethodReference
 
 @Patch(false)
 @Name("remove-screenshot-restriction")
-@Description("Removes the restriction of making screenshots.")
+@Description("Removes the restriction of making screenshots in some apps that prevent it due to security/piracy reasons. This patch is universal patch, meaning that you can apply it on every apps.")
```

Comment:
I dont think it is necessary to randomly explain here what a universal patch is. The description should be specific to the patch, but now it explains what a universal patch is and what you can do with it, out of topic. Additionally the first sentence does not add any more explanation on what the patch does. The patch removed the restriction to make screenshots in apps. What the restriction is for does not matter for the patch because it explains why the restriction exists and not what the patch does.